homeassistant/helpers/config_validation.py

```diff
@@ -876,8 +879,15 @@ def key_value_validator(value: Any) -> dict[Hashable, Any]:
         if isinstance(key_value, Hashable) and key_value in value_schemas:
             return cast(Dict[Hashable, Any], value_schemas[key_value](value))
 
+        if default_schema:
+            with contextlib.suppress(vol.Invalid):
+                return cast(Dict[Hashable, Any], default_schema(value))
```

Comment:
This will hide error messages from the default validator, not sure if that's acceptable?